### ISSUE TITLE
Linked NPC groups aren't saved when restarting SynthEBD

### ISSUE THREAD

**Issue by leonardovcr** [Synthesis-Collective/SynthEBD#98]
Hey Piranha, another bug... this time quick and simple fortunately. Similar to issue #91, Linked NPC groups aren't saved when restarting SynthEBD. 

Edit.: Oh, and Linked NPC Name Exclusions too...

Edit2.: So investigating a bit more, I noticed that there are three related files for that. 
 - GeneralSettings.json
 - LinkedNPCs.json
 - LinkedNPCNameExclusions

GeneralSettings.json contains both LinkedNPCs and LinkedNPCNameExclusions. Maybe that's old code which should have gone to those last two files?

I think SynthEBD is reading/writing from/to GeneralSettings.json, since if I add a new LinkedNPCNameExclusions, it persists on that .json file, but not on LinkedNPCNameExclusions.json. If I add a new Linked NPC group, it's not persisted in both files (LinkedNPCs.json and GeneralSettings.json).

**Comment by Piranha91** [comment]
Whoops, this is actually a new bug I introduced in 0.9.4.3 - Find & Replaced missed the Linked NPC Groups for some reason and I didn't notice. Fixed in 0.9.4.5. I couldn't replicate the Linked NPC Name Exclusions - those saved and reloaded fine for me.

Both features now live in GeneralSettings.json as of v0.9. The two loose json files are no longer used (probably left over from overwriting a previous SynthEBD installation in the same folder).
